src/commands/bootcamp/bootcampKickMentee.ts

```diff
@@ -0,0 +1,48 @@
+import { Message, VoiceChannel, TextChannel } from 'discord.js';
+import { CommandoClient, CommandoMessage } from 'discord.js-commando';
+import { MentorCommand } from '../../utils/commands';
+
+class BootcampKickMenteeCommand extends MentorCommand {
+  constructor(client: CommandoClient) {
+    super(client, {
+      name: 'kick',
+      aliases: ['remove', 'boot'],
+      group: 'bootcamp',
+      memberName: 'kick',
+      guildOnly: true,
+      description: 'Kick mentees out of your 1 on 1 call.'
+    });
+  }
+
+  async onRun(message: CommandoMessage): Promise<Message> {
+    const mentor = message.member;
+    const callChannel = mentor?.voice?.channel;
+    const guild = message.guild;
+    const track = callChannel?.parent?.name;
+
+    const queueVoice = <VoiceChannel>(
+      guild.channels.cache.find((channel) => channel.name === track && channel.type === 'voice')
```

Comment:
there should be a constant for `voice` in the discord.js lib, we should use taht instead